src/components/UrlUploader.tsx

```diff
@@ -36,7 +36,7 @@ const UrlUploader: FC<Props> = ({ setFiles, setView }: Props) => {
             />
             <button
                 className="bg-blue-500 text-white p-2 w-full mt-2 rounded-md hover:bg-blue-600 active:bg-blue-700 transition-all duration-300"
-                type="button"
+                type="submit"
```

Comment:
this will trigger form submissions

Reply:
> this will trigger form submissions

Yup, will trigger the parent form of uNotes